specs/order_spec.rb

```diff
@@ -78,32 +79,49 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
+  before do
+    @orders = Grocery::Order.all
+  end
```

Comment:
Good use of a `before` block here to DRY up this code, and make sure it's consistent.